specs/host_list_view.adoc

```diff
@@ -0,0 +1,146 @@
+// vim: tw=79
+
+= Tendrl Definitions: Display list of hosts discovered by Tendrl
+
+Display list of hosts discovered by Tendrl, whether they currently associated with a cluster or not. The list should have functionality for 
+
+* filtering by name, cluster, role and status.
+* sorting by name.
+
+Host list will be sorted by name by default.
+
+== Problem description
+
+Tendrl should show list of hosts discovered by Tendrl along with their details.
+
+The details shown for each host in UI is -
+
+* Host Name
+* Status
+* IP Address
+* Storage Usage
+* CPU Utilisation
+* Memory Usage
+* Cluster Name
+* Role
+* Alert Count
+
+API support present for -
+
+* Host Name
+* IP Address
+* Storage Usage
+* CPU Utilisation
+* Memory Usage
+* Alert count
+
+== Use Cases
+
+Users can see the list of host present in Tendrl. They can also 
+
+* filter by name, cluster, role and status.
+* sort by name.
+
+As per the API support, users can 
+
+* filter by name, cluster
+* sort by name
+
+== Proposed change
+
+* Create a view for listing host details.
+* For showing storage usage, cpu utilisation and memory usage, use donut chart component.
+* Implement client side filer and sorting functionality.
+* Filter call will trigger after 1 sec the user enters the filter data.
+* Send polling call after 10 mins to get the updated data. 
+
+=== Alternatives
+
+Generate list component can be created and used for listing the host. But because of its complexity and time constraint, we will pick this up later.
+
+=== Data model impact:
+
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+/api/1.0/GetNodeList
+
+Required Input from Anup - 
+
+* To add the response format for the API call
+* To add the polling API call and its response format	
+
+==== Notifications/Monitoring impact:
+
+Required Input from Anmol: To add the response format for monitoring data
```

Comment:
@gnehapk The details that need to go here are captured as part of https://github.com/Tendrl/specifications/pull/81 please reference this from here.